### PR TITLE
Feature/adding ideas pt2

### PR DESCRIPTION
**Pull request by brendel-r**
We added functionality to the clear  form on button click. We also added the input text to cards on button click.

### CODE REVIEW

File: Idea.js [hunk 1]
```diff
@@ -9,5 +9,4 @@ class Ideas {
 
     }
 }
-
-module.exports = Ideas;
```
Comment (asutherland91) [inline Idea.js:13]: I wonder why we don't need this

File: main.js [hunk 2]
```diff
@@ -18,10 +17,17 @@ var formSearch = document.querySelector("#search-form");
 saveButton.addEventListener("click", saveIdea);
 
 //functions:
-// we need a save function
-    // needs to be able to update the card title and body input field
-function saveIdea(){
-    var newCard = new Ideas(formTitle.value, formBody.value) {
+function saveIdea(event) {
```
Comment (asutherland91) [inline main.js:20]: This section looks really good, It's a little big I wonder if we can make it smaller

**Review by RandyGitProjects — COMMENTED** [review]
Do we still need module.exports? The function for the save button has dry code. Curious if we could make another function to make it drier for lines 23/24.

File: main.js [hunk 3]
```diff
@@ -18,10 +17,17 @@ var formSearch = document.querySelector("#search-form");
...
+    var newCard = new Ideas(formTitle.value, formBody.value);
+    ideaTitle.innerText = newCard.title;
+    ideaBody.innerText = newCard.body;
+    ideas.push(newCard);
+    clearInputFields();
+}
+
+function clearInputFields(){
+    formTitle.value = '';
+    formBody.value = '';
```
Comment (Denzel-Turambi) [inline main.js:31]: very clean and dry!

**Pull request merged**